env/#composite_env.cc#

```diff
@@ -0,0 +1,464 @@
+// Copyright (c) 2019-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
```

Comment:
Do you create another copy of "env/#composite_env.cc# "?

Reply:
Accidental file add.  Removed.